cli/tokens.go

```diff
@@ -67,7 +67,7 @@ func createToken() *cobra.Command {
 			cmd.Println(cliui.Styles.Code.Render(strings.TrimSpace(res.Key)))
 			cmd.Println()
 			cmd.Println(cliui.Styles.Wrap.Render(
-				fmt.Sprintf("You can use this token by setting the --%s CLI flag, the %s environment variable, or the %q HTTP header.", varToken, envSessionToken, codersdk.SessionTokenKey),
+				fmt.Sprintf("You can use this token by setting the --%s CLI flag, the %s environment variable, or the %q HTTP header.", varToken, envSessionToken, codersdk.SessionCustomHeader),
```

Comment:
I discovered that `coder_session_token` (SessionTokenKey) doesn't work as a header, but `Coder-Session-Token` (SessionCustomHeader) works.

Reply:
Great catch, yes this is my mistake and you are correct here.